core/src/main/scala/org/apache/spark/rdd/PairRDDFunctions.scala

```diff
@@ -70,12 +71,13 @@ class PairRDDFunctions[K, V](self: RDD[(K, V)])
    * In addition, users can control the partitioning of the output RDD, and whether to perform
    * map-side aggregation (if a mapper can produce multiple items with the same key).
    */
-  def combineByKey[C](createCombiner: V => C,
+  def combineByKeyWithClassTag[C](
```

Comment:
actually, why call this something else? Does it not compile if you just called this `combineByKey` as well?